application.js

```diff
@@ -1,16 +1,25 @@
 var mbaasApi = require('fh-mbaas-api');
 var express = require('express');
 var mbaasExpress = mbaasApi.mbaasExpress();
-var cors = require('cors');
+//var cors = require('cors');
+var proxy = require('express-http-proxy');
 
 // list the endpoints which you want to make securable here
 var securableEndpoints;
 securableEndpoints = [];
 
 var app = express();
 
+var jbpmPort = process.env.JBPM_PORT || process.env.OPENSHIFT_JBPM_PORT || 8080;
```

Comment:
No need for the fallback to `process.env.OPENSHIFT_JBPM_PORT` here - can remove these, and just go with:

```
process.env.JBPM_PORT || 8080;
```

Same for the rest below. 

Those old `process.env.OPENSHIFT_NODEJS_PORT` were needed for OpenShift V2, which we don't really support anymore.